libcodechecker/libhandlers/store.py

```diff
@@ -308,6 +308,7 @@ def get_analysis_statistics(inputs, limits):
     Collects analysis statistics information and returns them.
```

Comment:
Please extend the function documentation a bit, what is returned a list of files to be packed into the zip?
In that case shouldn't we return an empty list if we do not want to collect anything.